### DISCUSSION TITLE
无法登录，出现提示:本次登录环境存在风险, 需使用手机号进行验证或绑定

### DISCUSSION THREAD

**Discussion by tpemail** [lifegpc/bili#110] [General]
登录时出现错误提示"本次登录环境存在风险, 需使用手机号进行验证或绑定"
复制出现的网址后要求手机验证，但验证后按回车键依然出现相同错误无法登录

**Comment by lifegpc** [comment]
请使用浏览器登录，Chrome去[这](https://chromedriver.chromium.org/)下个与你Chrome大版本一致的chromedriver，解压后把exe丢到程序目录即可
如果是Firefox，下一个geckodriver，解压后同样丢到程序目录

**Reply by myishh** [reply]
> 请使用浏览器登录，Chrome去[这](https://chromedriver.chromium.org/)下个与你Chrome大版本一致的chromedriver，解压后把exe丢到程序目录即可 如果是Firefox，下一个geckodriver，解压后同样丢到程序目录

为什么这么做呢
